Resources/public/templates/fields/view/float.hbt

```diff
@@ -0,0 +1,6 @@
+<div class="ez-fieldview-row pure-g">
+    <div class="ez-fieldview-label pure-u">
+        <p class="ez-fieldview-name"><strong>{{ fieldDefinition.names.[eng-GB] }}</strong></p>
```

Comment:
is the ```eng-GB``` hardcoded for some reason? (asking from my ignorance about the whole thing)